### DISCUSSION TITLE
0.4.1

### DISCUSSION THREAD

**Discussion by kubilus1** [kubilus1/autoortho#93] [Announcements]
## What's Changed
* Mm0 partial + 2 bug fixes by @hotbso in https://github.com/kubilus1/autoortho/pull/54
* Default to DXT1 format compression by @kubilus1 in https://github.com/kubilus1/autoortho/pull/66
* Default to Dokan with FUSE.  Allow winfsp raw mode as an override.  Check for libs. by @kubilus1 in https://github.com/kubilus1/autoortho/pull/75
* Integrate Aoimage from @hotbso by @kubilus1 in https://github.com/kubilus1/autoortho/pull/74
* Update winsetup.py by @marcfsx in https://github.com/kubilus1/autoortho/pull/79
* Fix when winfsp is not present by @kubilus1 in https://github.com/kubilus1/autoortho/pull/80
* Support partial compression of mipmap 0 by @kubilus1 in https://github.com/kubilus1/autoortho/pull/58
* Add max wait limit for chunks.  Use lower quality chunks as a fallback. by @kubilus1 in https://github.com/kubilus1/autoortho/pull/92
* A few tweaks by @kubilus1 in https://github.com/kubilus1/autoortho/pull/87

## New Contributors
* @hotbso made their first contribution in https://github.com/kubilus1/autoortho/pull/54
* @marcfsx made their first contribution in https://github.com/kubilus1/autoortho/pull/79

**Full Changelog**: https://github.com/kubilus1/autoortho/compare/0.3.1...4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kubilus1/autoortho/releases/tag/4.1'>4.1</a>.</em>

**Comment by ivotaHR** [comment]
thank you

**Reply by kubilus1** [reply]
You are welcome!

**Comment by kubilus1** [comment]
Main features of this release over 4.0 is that it includes a maximum wait time for data retrieval in order to keep things running smoothly.

**Comment by Jeff2122** [comment]
@kubilus1  Loving AO and use it daily. I wonder what, if any, messages that scroll across the server screen would you llke to see? Diagnostics-wise I mean.
I am seeing a lot of "already retrieved. Exit" messages and "empty chunk data. Skip" "packet timed out. NBD." and sometimes other stuff. The sim does load eventually and everything seems fine at first, but some of the textures are extremely low resolution below me as I fly  (I fly <FL060 when possible). I can upload whatever might help.

**Reply by kubilus1** [reply]
Those message are just informational, and don't mean anything is wrong necessarily.  If you mostly fly low and slow and you want to avoid the occasional low quality texture you can increase the 'maxwait' time in your .autootho config file.

**Reply by Jeff2122** [reply]
Thanks, will try that. Ok on the info, If I see errors I'll ask again.

**Reply by darkpino** [reply]
Where can I find the .autootho config file?

**Reply by ivotaHR** [reply]
C:\Users\ #your_profil_name# \ .autoortho

**Reply by darkpino** [reply]
Thanks for the quick reply!

**Comment by Jeff2122** [comment]
The increase worked. I made it 3 seconds. I now have no ugly squares. I also don't have stutters as others are reporting. I am flying in Europe mostly right now, using Bing (no override).

**Comment by darkpino** [comment]
Is there any cure for the GETORTHO:FAILED error? I really don’t understand why that’s happening

**Reply by kubilus1** [reply]
You are hitting free map providers so they do limit a bit how fast you can fetch stuff.  But your ISP can also do dirty tricks on you, or maybe your network is a bit flakey, etc.  Hard to say.

**Reply by darkpino** [reply]
But how come that for example downloading over usenet doesn’t give any troubles and this does? And it doesn’t seem to make a difference which provider I choose so I really don’t understand what’s going on

**Reply by kubilus1** [reply]
The difference is that it's entirely different.  I'm assuming by 'downloading over usenet' you are talking about probably a few large files.  With these satellite map APIs you are doing many many specific calls.  Furthermore, the server side and ISP side can control a lot of this.  Are you using your ISPs DNS provider?  That could be part of it as well.  Likely works better on Linux as well, due to built in DNS caching, etc.

**Comment by Awemeter** [comment]
The issue of no ortho loadings has returned. All I see are stars.
